README.md

```diff
@@ -6,295 +6,354 @@
-for more details).
-
-_Notice that the `serviceClass`, `plan`, `serviceClassType`, and `externalName` fields are immutable. Immutability is not enforced
-with an admission controller, so updates go through initially successfully. However, the controller intercepts such changes and
-changes those fields back to their original values. So although it may seem that updates to those fields are accepted, they are
-in fact reverted by the controller. On the other hand, `parameters` and `tags` are updatable._
-
-The IBM Cloud Operator needs an account context, which indicates the `api-key` and the details of the IBM Public Cloud
-account to be used for service instantiation. The `api-key` is contained in a Secret called `secret-ibm-cloud-operator` that is created
-when the IBM Cloud Operator is installed. Details of the account (such as organization, space, resource group) are held in a
-ConfigMap called `config-ibm-cloud-operator`. To find the secret and configmap the IBM Cloud Operator first looks at the namespace of the
-resource being created, and if not found, in a management namespace (see below for more details on management namespaces). If there is no management namespace, then the operator looks for the secret and configmap in the `default` namespace. 
-
-
-The account information can be overriden by using
-the `context` field in the service yaml, with the following substructure:
-
-| Field            | Required | Type     |
-|:-----------------|:---------|:---------|
-| org              | No       | `string` |
-| space            | No       | `string` |
-| region           | No       | `string` |
-| resourceGroup    | No       | `string` |
-| resourceGroupID  | No       | `string` |
-| resourceLocation | No       | `string` |
-
-To override any element, the user can simply indicate that element and omit the others.
-If a `resourceGroup` is indicated, then the `resourceGroupID` must also be provided. This can be obtained with the
-following command, and retrieving the field `ID`.
+A `Binding` custom resources includes the properties in the following table. For more information, see the [IBM Cloud Operator user guide](docs/user-guide.md).
 
-```bash
-ibmcloud resource group <resourceGroup>
-```
+| Field            | Required | Type     | Comments                                                                                              |
+|:-----------------|:---------|:---------|:------------------------------------------------------------------------------------------------------|
+| serviceName      | Yes      | `string` | The name of the `Service` resource that corresponds to the service instance on which to create credentials for the binding. |
+| serviceNamespace | No       | `string` | The namespace of the `Service` resource.|
+| alias            | No       | `string` | The name of credentials, if this binding links to existing credentials.<!--what does this mean?-->|
```

Comment:
> `alias`: The name of credentials, if this binding links to existing credentials.

What does this mean?

Reply:
"alias" is used in a couple places in ICO. Basically it means that the operator will associate the custom resource with an existing IBM Cloud service or binding, but will not manage it's creation or deletion in IBM Cloud.

In this specific case, the alias is the name of the existing credentials in IBM Cloud. After creation, this Binding CR will create a secret for the existing binding and place it in the cluster.

Reply:
ok, I think I get it...I updated the description to: 
> The name of existing IBM Cloud credentials to link this binding to. This binding creates a secret for these credentials in the cluster namespace, but cannot modify the existing credentials in IBM Cloud.